websites/T/Trakt/presence.ts

```diff
@@ -0,0 +1,179 @@
+				document
+					.querySelector(
+						"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h2 > a:nth-child(1)"
+					)
+					.textContent.split(":")[0]
+			} - 
+		${
+			document.querySelector(
+				"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1 > span.main-title-sxe"
+			).textContent
+		}`;
+		} else if (path.includes("/seasons/")) {
+			presenceData.details = "Browsing season...";
+			presenceData.state = `${
+				document.querySelector("#level-up-link").textContent
+			}: ${
+				document.querySelector(
+					"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1"
+				).firstChild.textContent
+			}`;
+		} else {
+			try {
+				presenceData.details = "Browsing TV show...";
+				presenceData.state = document.querySelector(
+					"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1"
+				).firstChild.textContent;
+			} catch (e) {
+				presenceData.details = "Browsing...";
+				presenceData.state = "";
+			}
+		}
+
+		if (path.includes("/trending")) presenceData.state = "trending shows";
+		else if (path.includes("/popular")) presenceData.state = "popular shows";
+		else if (path.includes("/recommended"))
+			presenceData.state = "recommended shows";
+		else if (path.includes("/anticipated"))
+			presenceData.state = "anticipated shows";
+		else if (path.includes("/watched"))
+			presenceData.state = "the most watched shows";
+		else if (path.includes("/collected"))
+			presenceData.state = "the most collected shows";
+	} else if (path.includes("/movies")) {
+		try {
+			presenceData.details = "Browsing movie...";
+			presenceData.state = document
+				.querySelector(
+					"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1"
+				)
+				.textContent.split(
+					document.querySelector(
+						"#summary-wrapper > div.container.summary > div > div > div.col-md-10.col-md-offset-2.col-sm-9.col-sm-offset-3.mobile-title > h1 > span"
+					).textContent
+				)[0];
+		} catch (e) {
+			presenceData.details = "Browsing...";
+			presenceData.state = "";
+		}
+
+		if (path.includes("/trending")) presenceData.state = "trending movies";
+		else if (path.includes("/popular")) presenceData.state = "popular movies";
+		else if (path.includes("/recommended"))
+			presenceData.state = "recommended movies";
+		else if (path.includes("/anticipated"))
+			presenceData.state = "anticipated movies";
+		else if (path.includes("/watched"))
+			presenceData.state = "the most watched movies";
+		else if (path.includes("/collected"))
+			presenceData.state = "the most collected movies";
+		else if (path.includes("/boxoffice"))
+			presenceData.state = "top 10 grossing movies";
```

Comment:
I'm deja vu of these lines, why are they needed twice?

Move L132 and L133 above to the other block

Reply:
its for movies and tv shows